joomla/joomla-cms
# [imp] Added a validation message for the URL field when schema is not provided

Pull Request for Issue #13870.

### Summary of Changes
Added a validation message to the JFormRuleUrl class.

### Testing Instructions
1. Create a url field : fill schema and make sure it's not relative
2. Create a new content and fill the url field with a domain name (example.com).
### Expected result
A warning message displays "Invalid URL : URL schema is missing in example.com. Please add one of the following at the begining : http...." with no explanation.
### Actual result
A warning message displays "Invalid field" with no explanation.
### Documentation Changes Required
No.
